src/utils/download.utils.ts

```diff
@@ -44,20 +60,28 @@ function downloadResource<T>(
   ) => void
 ): Promise<T> {
   return new Promise<T>((resolve, reject) => {
-    const parsedUrl = url.parse(urlStr);
-    const { agent, rejectUnauthorized } = getHttpsProxyAgent();
-    const request: ClientRequest = getHttpClient(parsedUrl).request(
+    const parsedUrl = new URL(urlStr);
+    const request: ClientRequest = https.request(
```

Comment:
this won't work in  dev environment where you'r running a localhost  selfhosted service

Reply:
you need the  https://github.com/codota/tabnine-vscode/pull/1311/files#diff-d99f2ff5ebc521a8a2442af204589a4f49f486fef88b68e564d345b37ca0ad36L99